integration-tests-ex/cases/cluster/Common/dependencies.yaml

```diff
@@ -46,28 +46,27 @@ services:
   kafka:
     image: bitnami/kafka:${KAFKA_VERSION}
     container_name: kafka
+    # platform: linux/x86_64
     labels:
       druid-int-test: "true"
     ports:
       - 9092:9092
-      - 9093:9093
+      - 9094:9094
```

Comment:
why the port change?

Reply:
i was having trouble getting the docker kafka with zookeeper working so i switched to using no zookeeper, which runs a controller on 9093, see https://github.com/bitnami/containers/blob/main/bitnami/kafka/README.md#using-a-docker-compose-file

Reply:
Ah, thanks for the clarification! Maybe we can include this info in the IT readme or somewhere in a follow up PR.